src/App.js

```diff
@@ -1,29 +1,9 @@
-import logo from './logo.svg';
-import './App.css';
+// import logo from './logo.svg';
+// import './App.css';
```

Comment:
- [ ] Remove all the commented codes

Reply:
- [ ] In the `src` folder I suggest that you keep files you are using only you don't need like, you can keep only `css` file you are using only
![ff](https://user-images.githubusercontent.com/44745855/148260680-dfe8b036-4c12-4549-9819-e5982cb08478.png)

- [ ] In the `public` folder you can just keep `index.html` only that is what you want for now